### PR TITLE
feat: powershell installation script

### PR DESCRIPTION
**Pull request by Xlin123**
**- What I did**
  Created a powershell installation script for sshnpd and sshnp (device and client side)
  
  For the device side I'm installing sshnpd as a windows service using winsw. For this, I need a xml file in the bundle as well, look  at sshnpd_service.xml. 
  
  For client side I'm making a mini script for them to execute on.
  
**- How I did it**

**- How to verify it**
 Please look at the readme, I'm not quite sure if this is how we want to install the client or not (I'm pretty sure it's outdated.)

For testing please use from the root of the repo.
.\packages\dart\sshnoports\bundles\universal.ps1 -dev
 
**- Description for the changelog**
feat: powershell installation script

### CODE REVIEW

**Comment by cconstab** [comment]
Couple of small things I noticed in the code before it gets merged.

The README.md needs updating

The version of sshnp needs updating

The idea and code looks solid but I have not treied it yet! 

Probably also needs an uninstall option..

**Comment by Xlin123** [comment]
nvm I just added a flag for repo testing

File: packages/dart/sshnoports/bundles/windows/universal.ps1
Comment (xavierchanth) [inline packages/dart/sshnoports/bundles/windows/universal.ps1:1]: This file should be moved up one level, we don't need to ship it in the bundle since it will be downloaded in advance

File: packages/dart/sshnoports/bundles/universal.ps1
Comment (TylerTrott) [inline packages/dart/sshnoports/bundles/universal.ps1]: Hm, backslash? Could be interesting problem :) 
```suggestion
        Invoke-Expression "$script:service_path stop" -ErrorAction SilentlyContinue
```

**Review by TylerTrott — CHANGES_REQUESTED** [review]

File: packages/dart/sshnoports/bundles/universal.ps1
Comment (Xlin123) [reply packages/dart/sshnoports/bundles/universal.ps1]: resolved in [b354179](https://github.com/atsign-foundation/noports/pull/916/commits/b354179539a1b787943860f126b6393a4d1b899e)

**Review by TylerTrott — DISMISSED** [review]
lgtm

**Comment by Xlin123** [comment]
I'd like to get everyones approval before we merge.

**Review by JeremyTubongbanua — DISMISSED** [review]

**Review by JeremyTubongbanua — DISMISSED** [review]

**Comment by xavierchanth** [comment]
I will clone a fresh windows vm and run later today

File: packages/dart/sshnoports/bundles/windows/README.md
Comment (xavierchanth) [inline packages/dart/sshnoports/bundles/windows/README.md:46]: nit: canonical paths in windows are with `\`

**Review by xavierchanth — APPROVED** [review]

**Pull request merged**
